github/config.go

```diff
@@ -23,9 +23,7 @@ func init() {
 		if u, err := user.Current(); err == nil {
 			homeDir = u.HomeDir
 		}
-	}
-
-	if homeDir == "" {
```

Comment:
It can only be shortened that way if `y` never changes in `a` block. During the first conditional block, does `homeDir` has a chance to change?